spk/sabnzbd/src/wizard/install_uifile

```diff
@@ -1,19 +1,31 @@
-[{
+[
+  {
     "step_title": "Basic configuration",
-    "items": [{
+    "items": [
+      {
         "type": "textfield",
-        "desc": "Download directory",
-        "subitems": [{
+        "desc": "Download configuration",
+        "subitems": [
+          {
             "key": "wizard_download_dir",
-            "desc": "Download directory",
+            "desc": "Download location",
             "defaultValue": "/volume1/downloads",
             "validator": {
-                "allowBlank": false,
-                "regex": {
-                    "expr": "/^\\\/volume[0-9]+\\\//",
-                    "errorText": "Path should begin with /volume?/ with ? the number of the volume"
-                }
+              "allowBlank": false,
+              "regex": {
+                "expr": "/^\\\/volume\\w*[0-9]{1,2}\\\/[^<>: */?\"]*/",
```

Comment:
Please relax regular expression to support `/usbshareX/` volumes according to #3038

Reply:
It is already relaxed, it now has the extra `\w`. Because for USBs the path is: `/volumeUSB1/usbshare1`.
I also put that in the installation message because for new users it's pretty confusing: 
https://github.com/SynoCommunity/spksrc/pull/3065/files#diff-2d2834ae3fa939e71e1208d004e4ec56R24

Reply:
Strange. My DSM mounts USB devices with `/usbshareX/` and comment from Toxine68 confirms too: https://github.com/SynoCommunity/spksrc/issues/3038#issuecomment-354251192
Reference documentation: https://www.synology.com/en-uk/knowledgebase/DSM/help/DSM/AdminCenter/system_externaldevice_devicelist
Where have you heard about `/volumeUSBx` ?

Reply:
Yes the _share_ is called `usbshare1`, but when you actually SSH into the device you will see that the path is `/volumeUSB1/usbshare1`. At least on my Xpenology. 
There is no `/usbshare1/` (without prefix).

Reply:
OK, my mistake, only looking at DSM file manager. I should have checked on file system.